packages/relay/src/lib/eth.ts

```diff
@@ -657,9 +657,12 @@ export class EthImpl implements Eth {
     try {
       const gasPrice = await this.getFeeWeibars(EthImpl.ethSendRawTransaction);
       await this.precheck.sendRawTransactionCheck(transaction, gasPrice);
-    }
-    catch(e: any) {
-      return e;
+    } catch (e: any) {
+      if (e instanceof JsonRpcError) {
+        return e;
+      }
+
+      throw predefined.INTERNAL_ERROR;
```

Comment:
There should be an `eth.spect.ts` test to highlight the issue the bug highlighted